test/main.spec.js

```diff
@@ -0,0 +1,14 @@
+import test from 'tape'
+import addTwoNumbers from './../src/main'
```

Comment:
To simplify a bit this, we can add `src` to the `NODE_PATH`.

See https://github.com/sphereio/sphere-node-sdk/blob/rewrite-2.0/package.json#L39